addons/phantom_camera/scripts/phantom_camera_host/phantom_camera_host.gd

```diff
@@ -509,8 +509,19 @@ func _assign_new_active_pcam(pcam: Node) -> void:
 					var pyramid_shape_data = Engine.get_singleton("PhysicsServer3D").call("shape_get_data",
 						camera_3d.get_pyramid_shape_rid()
 					)
+
+					# Scale up the pyramid shape to avoid clipping issues
+					var expanded_points := PackedVector3Array()
+					for point in pyramid_shape_data:
+						var expanded_point := Vector3(
+							point.x + (0.02 if point.x >= 0 else -0.02),
+							point.y + (0.02 if point.y >= 0 else -0.02),
+							point.z + (0.02 if point.z >= 0 else -0.02)
```

Comment:
I played with this value a bit and `0.01` was too small to avoid the clipping, `0.02` seems to work in most cases.

Also not sure if this should be configurable or not - personally, I think this is a sane default.